doc/known-issues.md

```diff
@@ -57,10 +57,10 @@ LD_PRELOAD=/run/current-system/profile/lib/libvulkan.so zig-out/bin/gpu-hello-tr
 
 **Background**: `zig build` on the first time around will download a 20-30MB file of Dawn (Google's WebGPU implementation) from https://github.com/hexops/mach-gpu-dawn/releases - the build system uses `curl` to do this automatically. You can of course build Dawn from source using the `-Ddawn-from-source=true` flag, but this will require a clone of the Dawn sources which are a larger download and takes a few minutes to build as it is a large C++ codebase.
 
-Users in Chinese mainland may find that download speeds to github.com are too slow (hours to download a 30 MB file), and apparently it is common to use GitHub mirror sites like https://fastgit.org to download files from GitHub.
+Users in Chinese mainland may find that download speeds to github.com are too slow (hours to download a 30 MB file), and apparently it is common to use GitHub mirror sites like https://hub.yzuu.cf to download files from GitHub.
 
 You can have Mach use such websites by setting an environment variable e.g.:
 
 ```sh
-MACH_GITHUB_BASE_URL=https://download.fastgit.org
+MACH_GITHUB_BASE_URL=https://hub.yzuu.cf
```

Comment:
With fastgit.org, there is some ability for me to confirm it is commonly used by people, that it appears to be a legitimate company, etc.

I am afraid to suggest anyone use `hub.yzuu.cf` because I can't verify this is a legitimate website; I don't know whether it could serve malware in the future, etc.

According to fastgit.org they have a new mirror: `hub.fastgit.xyz` - I am happy to update to that one here.